packages/frint-props/README.md

```diff
@@ -0,0 +1,384 @@
+# frint-props
+
+[![npm](https://img.shields.io/npm/v/frint-props.svg)](https://www.npmjs.com/package/frint-props)
+
+> Compose reactive props for FrintJS Apps
+
+<!-- MarkdownTOC autolink=true bracket=round -->
+
+- [Guide](#guide)
+  - [Installation](#installation)
+  - [Concepts](#concepts)
+  - [Usage](#usage)
+  - [Write your own functions](#write-your-own-functions)
+- [API](#api)
+  - [withDefaults](#withdefaults)
+  - [withState](#withstate)
+  - [withState](#withstate-1)
```

Comment:
Why is `withState` linked twice? It seems like a typo somewhere between `withDefaults` and `withState` descriptions.

Reply:
investigating... this block is generated automatically by SublimeText's MarkdownTOC plugin. not sure why it thinks there are two `withState`s.

Reply:
well.. I did add two `withState`s myself. removing it...